src/ribasim_nl/ribasim_nl/__init__.py

```diff
@@ -1,9 +1,15 @@
 __version__ = "0.1.0"
 
+import warnings
+
 from ribasim_nl.cloud import CloudStorage
 from ribasim_nl.model import Model
 from ribasim_nl.network import Network
 from ribasim_nl.network_validator import NetworkValidator
 from ribasim_nl.reset_index import reset_index
 
 __all__ = ["CloudStorage", "Network", "reset_index", "Model", "NetworkValidator"]
+
+
+# ignore crs-warning in Ribasim-module, see https://github.com/Deltares/Ribasim/issues/1799
+warnings.filterwarnings("ignore", message="CRS not set for some of the concatenation inputs", category=UserWarning)
```

Comment:
Linked issue is fixed, and Basin / area will be resolved today: https://github.com/Deltares/Ribasim/pull/1781/commits/71d9fbc3a5a2fadd5698c1a3c56da070adf0619f

I don't want Ribasim-NL to silence any errors that should be fixed upstream.